Automattic/jetpack
# Search: Backport Customberg's Babel configuration from WPCOM

Follow-up for #20369.

#### Changes proposed in this Pull Request:
* Updates Customberg's Babel configuration to use that of Instant Search.
(This change has already been deployed via r230498-wpcom and is being backported to Jetpack.)

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your Jetpack Search installation on WordPress 5.8.
* Navigate to the new customization page at `/wp-admin/admin.php?page=jetpack-search-customize`.
* Ensure that the page loads and works as expected.
